src/groups/special_orthogonal.jl

```diff
@@ -14,20 +14,47 @@ end
 
 # optimized inverseion for the special orthogonal group
```

Comment:
This comment doesn't apply here after this change, it should be removed or moved to `inv(G::SpecialOrthogonal, x)`